packages/builder/src/stores/builder/navigation.ts

```diff
@@ -29,10 +58,18 @@ export class NavigationStore extends BudiStore<AppNavigationStore> {
   }
 
   async save(navigation: AppNavigation) {
-    const appId = get(appStore).appId
-    const app = await API.saveAppMetadata(appId, { navigation })
-    if (app.navigation) {
-      this.syncAppNavigation(app.navigation)
+    if (!featureFlag.isEnabled(FeatureFlag.WORKSPACE_APPS)) {
```

Comment:
Maybe a silly question but how come this doesn't use `$featureFlags` like above in the constructor?

Reply:
`$featureFlags` is a store, so it's not accessible. We could use `get(featureFlags)` instead, but it would be the same as reactivity would not apply either